DataFormats/L1TParticleFlow/interface/datatypes.h

```diff
@@ -43,6 +43,7 @@ namespace l1ct {
   typedef ap_ufixed<10, 5, AP_TRN, AP_SAT> hoe_t;
   typedef ap_uint<4> redChi2Bin_t;
   typedef ap_fixed<10, 1, AP_RND_CONV, AP_SAT> id_score_t;  // ID score to be between -1 (background) and 1 (signal)
+  typedef ap_ufixed<10, 1, AP_RND, AP_SAT> b_tag_score_t; // result_t from the NN is still apx_fixed<16,6>
```

Comment:
I think `ap_ufixed<N,1>`  has a range between 0 (included) and 2 (excluded), which is a bit strange for a discriminator.

Reply:
Ah yea I see, this should have been `typedef ap_ufixed<10, 0, AP_RND, AP_SAT> b_tag_score_t` I think. So then it would be between 0 (included) and 1 (excluded).  

I don't really know if this is what the GT people want ... so this is just something I picked. I think we need to sit down at some point with GT team to determine what is the best way to package the tagging scores.

Reply:
@gpetruc I just talked to @thesps and I think we want to have the option to inlcude 1 as the tagging score. I restored the former definition using `ap_ufixed<10, 1>` now